src/qtism/data/content/BodyElement.php

```diff
@@ -272,4 +329,290 @@ public function getDir()
     {
         return $this->dir;
     }
+
+    /**
+     * @param string $ariaControls
```

Comment:
Not really, it will throw an `InvalidArgumentException` that is why it's not in the PhpDoc. It's the same for the various comments you left so I will resolve them.